src/main/java/de/retest/recheck/review/ignore/io/Loader.java

```diff
@@ -1,11 +1,21 @@
 package de.retest.recheck.review.ignore.io;
 
+import java.util.Optional;
+import java.util.stream.Stream;
+
 public interface Loader<T> {
 
-	boolean canLoad( final String line );
+	Optional<T> load( String line );
 
-	T load( String line );
+	default Stream<T> load( final Stream<String> lines ) {
+		return lines.map( this::load ) //
```

Comment:
Sorry, looks like my comment got lost or I've forgot to send … you can use our util here:

```java
return lines.map( this::load ).flatMap( StreamUtil:optionalToStream );
```